src/Api/reviewPoster.ts

```diff
@@ -0,0 +1,9 @@
+import api from './api';
+
+const SPECIFIED_CATEGORY_ID = '63bd141d93836272216d324a';
+
+export const getSpecifiedReviewPoster = () =>
+  api.get(`/posts/channel/${SPECIFIED_CATEGORY_ID}`);
```

Comment:
generic적절히 사용하면 사용처에서 타입추론 가능할것 같네요

```suggestion
export const getSpecifiedReviewPoster = <T>(): Promise<AxiosResponse<T, any>> =>
  api.get<T>(`/posts/channel/${SPECIFIED_CATEGORY_ID}`);
```

![image](https://user-images.githubusercontent.com/30822283/212852843-4e3da502-6dec-4bde-b774-00cc8e80ca39.png)

**사용처**
as-is
![image](https://user-images.githubusercontent.com/30822283/212853115-104bfd08-9dde-49df-a671-1c6b85d2816c.png)

to-be
![image](https://user-images.githubusercontent.com/30822283/212853124-21a19826-f502-4817-abf7-56da90ad6365.png)

Reply:
p2